extensions/vscode/src/commands/new-bloc.command.ts

```diff
@@ -29,10 +30,10 @@ export const newBloc = async (uri: Uri) => {
     targetDirectory = uri.fsPath;
   }
 
-  const useEquatable = await hasDependency("equatable");
+  const useDependency = await getUsingDependency();
```

Comment:
```suggestion
  const dependency = await getUsingDependency();
```

This variable is no longer a boolean, so it should be named differently, in my opinion.
It made sense with `useEquatable` as it is a yes or no sort of question.